css/hero.css

```diff
@@ -0,0 +1,76 @@
+.hero {
+  background-color: #fff;
+  width: 375px;
+  height: 626px;
+  background-image: url("../images/header-shapes\ mobile@2x.svg");
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
- [ ] Your entire hero section only takes up a middle section of the available space and has spaces on both sides from `425px` 📱 and above. Please make it take up the whole width as shown in the given design
- [ ] Please make same changes to your navigation
### Your Design
![image](https://user-images.githubusercontent.com/46839548/164424774-25273d53-ed6a-4335-bdcf-256a84e84eea.png)

Reply:
I've just done it. Thank you for your help, Codepantha!

Reply:
You're welcome, @dqarias